spec/models/mode_of_trial_reason_spec.rb

```diff
@@ -0,0 +1,5 @@
+# require 'rails_helper'
+
+# RSpec.describe ModeOfTrialReason, type: :model do
+# pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Pref to put some basic test in here, to say check that the object is of the class defined, rather than merging commented code